docs/abtest.rst

```diff
@@ -71,7 +71,7 @@ Traffic Cop
 
 Create a .js file where you initialize Traffic Cop and include that in the
 experiments block in the template that will be doing the redirection. Wrap the
-extra js include in a `switch <http://bedrock.readthedocs.io/en/latest/install.html#feature-flipping-aka-switches>`_.
```

Comment:
this section heading was updated and no longer links directly to the section without a corresponding link update